sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -261,18 +264,194 @@ def _is_json_content_type(content_type: str) -> bool:
     return False
 
 
+def _build_sentinel(setting: Union[str, Tuple[str, str]]) -> Tuple[str, str]:
+    try:
+        key, label = setting
+    except IndexError:
+        key = setting
+        label = EMPTY_LABEL
+    if "*" in key or "*" in label:
+        raise ValueError("Wildcard key or label filters are not supported for refresh.")
+    return key, label
+
+
+def _is_retryable_error(error: HttpResponseError) -> bool:
+    """Determine whether the service error should be silently retried after a backoff period, or raised.
+    Don't know what errors this applies to yet, so just always raising for now.
+    :param error: The http error to check.
+    :type error: ~azure.core.exceptions.HttpResponseError
+    :return: Whether the error should be retried.
+    :rtype: bool
+    """
+    # 408: Request Timeout
+    # 429: Too Many Requests
+    # 500: Internal Server Error
+    # 504: Gateway Timeout
+    return error.status_code in [408, 429, 500, 504]  # Back off on throttling error
```

Comment:
Codes are already documented above
```suggestion
    return error.status_code in [408, 429, 500, 504]
```